rs/sns/cli/src/lib.rs

```diff
@@ -57,6 +59,19 @@ const TEST_NEURON_1_OWNER_DFX_IDENTITY_NAME: &str =
 pub struct CliArgs {
     #[clap(subcommand)]
     pub sub_command: SubCommand,
+
+    /// Override the compute network to connect to. By default, the local network is used.
+    /// A valid URL (starting with `http:` or `https:`) can be used here, and a special ephemeral
+    /// network will be created specifically for this request. E.g. "http://localhost:12345/"
+    /// is a valid network name.
+    // TODO: Remove this arguments once we can inherit the same data from dfx_core.
+    #[clap(long, aliases = ["ic-url"], default_value = "https://ic0.app")]
+    pub network: Option<String>,
+
+    /// Path to the PEM file of an identity to run this command as.
+    // TODO: Remove this arguments once we can inherit the same data from dfx_core.
```

Comment:
I've created a ticket for this

```suggestion
    // TODO(NNS1-3569): Remove this arguments once we can inherit the same data from dfx_core.
```